doc/howto/network_bridge_resolved.md

```diff
@@ -70,6 +70,60 @@ You must repeat the commands after each reboot and after LXD is restarted, or ma
 
 ## Make the `resolved` configuration persistent
 
+### Solution "A"
+
+You can automate the `systemd-resolved` DNS configuration, so that it is applied on system start and takes effect when LXD creates the network interface.
+
+To do so, create a `systemd` network file named `/etc/systemd/network/<network_bridge>.network` with the following content:
+
+```
+[Match]
+Name=<network_bridge>
+[Network]
+DNS=<dns_address>
+Domains=~<dns_domain>
+```
+
+So for example, `/etc/systemd/network/lxdbr0.network` with the following content (your IP, of course):
+
+```
+[Match]
+Name=lxdbr0
+[Network]
+DNS=10.167.146.1
+Domains=~lxd
+```
+
+If you have NOT freshly installed `lxd` (you've rebooted since install), then you only need to reload `systemd-resolved`:
```

Comment:
```suggestion
#### Apply the updated configuration

If you have rebooted since you first installed LXD, you only need to reload `systemd-resolved`:
```